src/voting/SimpleGaugeVoter.sol

```diff
@@ -181,7 +181,7 @@ contract SimpleGaugeVoter is
         voteData.lastVoted = block.timestamp;
     }
 
-    function reset(uint256 _tokenId) external nonReentrant whenNotPaused whenVotingActive {
+    function reset(uint256 _tokenId) external nonReentrant whenNotPaused {
```

Comment:
Are we not using the convention of 1_1_0 here? Later on this is going to get hard to trace if all other contracts work differently

Reply:
Moved changes to https://github.com/aragon/ve-governance/pull/126/files